src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+
+/**
+ * Duke is a chatbot that can perform specified commands requested by clients.
+ */
+public class Duke extends Application {
+    private Storage storage;
+    private Ui ui;
+    private TaskList tasks;
+
+    /**
+     * Constructor of Duke class
+     *
+     * @param filePath address of file path
+     */
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (DukeException e) {
+            //ui.showLoadingError();
+            tasks = new TaskList();
+        }
+    }
+
+    /**
+     * Runs Duke program
+     *
+     * @throws FileNotFoundException If the file path cannot be found
+     */
+    public void run() throws FileNotFoundException {
+        Scanner sc = new Scanner(System.in);
+        ui.showWelcome();
+        boolean isExit = false;
```

Comment:
I like the clear naming of boolean variable here. 👍

Reply:
Thanks for your compliment 👍